spec/tests/rules/format.json

```diff
@@ -101,6 +101,22 @@
     ]
   },
   {
+    "description": "validation of date-time strings",
```

Comment:
Strange there were no tests for this format at all. There is a mistake in date format tests above when in description it's called "date-time" - maybe that's why... Thank you